Monocle/Crawler/Provider.dhall

```diff
@@ -3,4 +3,5 @@
 | GithubProvider : ../Provider/Github/Type.dhall
 | GerritProvider : ../Provider/Gerrit/Type.dhall
 | BugzillaProvider : ../Provider/Bugzilla/Type.dhall
+| TaskDataProvider
```

Comment:
Why the TaskDataProvider is needed here ?

Reply:
This is for abritary task data, e.g. the existing lentille-bugzilla.